package.json

```diff
@@ -35,22 +35,18 @@
     "@storybook/addon-storysource": "^3.4.12",
     "@storybook/react": "~3.4.12",
     "@storybook/storybook-deployer": "^2.0.0",
-    "@theforeman/builder": "^3.5.2",
-    "@theforeman/env": "^3.5.2",
-    "@theforeman/vendor-dev": "^3.5.2",
+    "@theforeman/builder": "^3.7.0",
+    "@theforeman/env": "^3.7.0",
+    "@theforeman/vendor-dev": "^3.7.0",
     "argv-parse": "^1.0.1",
     "axios-mock-adapter": "^1.10.0",
     "babel-eslint": "^10.0.0",
-    "babel-jest": "^24.9.0",
     "babel-loader": "^8.0.0",
     "compression-webpack-plugin": "~1.1.11",
     "coveralls": "^3.0.0",
     "cross-env": "^5.2.0",
     "css-loader": "^0.23.1",
     "dotenv": "^5.0.0",
-    "enzyme": "^3.4.0",
-    "enzyme-adapter-react-16": "^1.4.0",
-    "enzyme-to-json": "^3.2.1",
```

Comment:
You will need to supply an alternative for `enzyme` and `enzyme-to-json` because they are imported from many places.
See `eslint` results:
```
  1:1  error  'enzyme-to-json' should be listed in the project's dependencies. Run 'npm i -S enzyme-to-json' to add it  import/no-extraneous-dependencies
  2:1  error  'enzyme' should be listed in the project's dependencies. Run 'npm i -S enzyme' to add it                  import/no-extraneous-dependencies
```

I would suggest doing something similar to:
- https://github.com/theforeman/foreman-js/pull/85/files#diff-d8567df8f65168c8284d7f590af7a553R88
- https://github.com/theforeman/foreman-js/pull/85/files#diff-b528f441e6e4b750b6df93564e94676cR1-R3